api/questions/sidewalk_search.py

```diff
@@ -9,30 +9,21 @@ def __init__(self, con):
         self.con = con
         
     def search_permits(self, search):
+        search = search[:50] # Limit to 50 characters
+        search = search.replace("_", "").replace("%", "").replace(";", "").replace("--", "") # Line 13 will remove these special characters 
```

Comment:
Can you add one more replace to replace single quotes `"'"` with an empty string `""`? That way, queries with an apostrophe won't cause an error. For example, `popeye's` will become `popeyes`.

<img width="735" alt="Screen Shot 2021-08-13 at 6 35 37 PM" src="https://user-images.githubusercontent.com/11896652/129427358-c4c58a2b-119e-42df-9e87-0209267b85d0.png">